lib/rpc/rpc.dart

```diff
@@ -161,9 +163,108 @@ class RPCLive implements RPC {
 
     return withdrawalTxid;
   }
+
+  @override
+  Future<List<Transaction>> listTransactions() async {
+    // first list
+    final transactionsJSON = await _client.call('listtransactions', []) as List<dynamic>;
+
+    // then convert to something other than json
+    List<Transaction> transactions = transactionsJSON.map((jsonItem) => Transaction.fromMap(jsonItem)).toList();
+    return transactions;
+  }
 }
 
 abstract class RPCError {
   static const errNoWithdrawalBundle = -100;
   static const errWithdrawalNotFound = -101;
 }
+
+class Transaction {
+  final String account;
+  final String address;
+  final String category;
+  final double amount;
+  final String label;
+  final int vout;
```

Comment:
do you write these out by hand, or do you have a tool for this?

Reply:
Have a tool, it’s called chatgpt

Reply:
The neatest tool of them all. I use no other tools for flutter